<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lbDispDate">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>2021/11/09</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbDispTime">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>30</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>11:12:13 AM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="Line" name="lnTopLine">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>421</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="grbODD">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>271</width>
      <height>121</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>ODD</string>
    </property>
    <widget class="QLCDNumber" name="lcdDigitNumber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>3</number>
     </property>
    </widget>
    <widget class="QLabel" name="lbDispUnit">
     <property name="geometry">
      <rect>
       <x>123</x>
       <y>70</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>km</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbDispLabelTop">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>21</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>12.C</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbDispLabelMiddle">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>54</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>40km/h</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbDispLabelBottom">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>87</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>100%</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpLampCtrl">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>421</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Control</string>
    </property>
    <widget class="QDial" name="diaLamp">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>121</width>
       <height>131</height>
      </rect>
     </property>
     <property name="maximum">
      <number>7</number>
     </property>
     <property name="pageStep">
      <number>10</number>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLampLow">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>36</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>LOW</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLampFront">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>77</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>FRONT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLampOff">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>159</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLampTail">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>118</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TAIL</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="icnState1">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>13</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icons.qrc">:/images/system/blank.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="icnState2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>13</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icons.qrc">:/images/system/blank.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="icnState4">
    <property name="geometry">
     <rect>
      <x>156</x>
      <y>13</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icons.qrc">:/images/system/blank.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="icnState3">
    <property name="geometry">
     <rect>
      <x>108</x>
      <y>13</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icons.qrc">:/images/system/blank.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="icnState5">
    <property name="geometry">
     <rect>
      <x>204</x>
      <y>13</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icons.qrc">:/images/system/blank.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLampEmergency">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Emergency</string>
    </property>
   </widget>
   <widget class="QDial" name="diaDirection">
    <property name="geometry">
     <rect>
      <x>97</x>
      <y>152</y>
      <width>121</width>
      <height>131</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLampDirOff">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>199</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLampDirRight">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>138</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>RIGHT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLampDirLeft">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>LEFT</string>
    </property>
   </widget>
   <widget class="QSlider" name="brakeSlider">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>26</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="accelSlider">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>26</width>
      <height>160</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>430</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="fuelPBar">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>300</y>
      <width>118</width>
      <height>24</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="hornBtn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>430</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hornBtn_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>340</y>
      <width>99</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QDial" name="wheelDial">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>210</width>
      <height>210</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
